helm/chaos-mesh/values.yaml

```diff
@@ -405,8 +405,14 @@ dnsServer:
   create: true
   # Name of serviceaccount for chaos-dns-server.
   serviceAccount: chaos-dns-server
-  # Image of DNS Server
-  image: ghcr.io/chaos-mesh/chaos-coredns:v0.2.6
+  # image would be constructed by <registry>/<repository>:<tag>
+  image:
+    # override global registry, empty value means using the global images.registry
+    registry: ""
+    # repository part for image of chaos-dashboard
+    repository: chaos-mesh/chaos-coredns
+    # override global tag, empty value means using the global images.tag
+    tag: ""
```

Comment:
it seems we should use the tag "v0.2.6" here.

chaos-coredns is a standalone component that does not get a new version released when Chaos Mesh core components release.

Reply:
rest LGTM!

and I do not prefer cherrypick it into the release-2.6, it contains a breaking changes in helm chart, it changes the schema of the field `dnsServer.image` from string to object

Reply:
Great, thank you for checking my PR. Your advise also LGTM!

Reply:
Hi @ChristianBieri1995, do you have time to update this one place? And, you also have to fix the DCO check and update the changelog.

Reply:
Updated, I will fix the DCO check later as soon as I have access to my local machine.